pkg/burner/create.go

```diff
@@ -149,7 +165,12 @@ func (ex *Executor) RunCreateJob(iterationStart, iterationEnd int) {
 		sem := make(chan int, int(ClientSet.RESTClient().GetRateLimiter().QPS())*2)
 		for i := iterationStart; i <= iterationEnd; i++ {
 			if ex.Config.NamespacedIterations {
-				ns = fmt.Sprintf("%s-%d", ex.Config.Namespace, i)
+				ns = fmt.Sprintf("%s-%d", ex.Config.Namespace, i/ex.Config.IterationsPerNamespace)
+				if namespacesWaited[ns] {
```

Comment:
do we need an `else` here?
```suggestion
				if namespacesWaited[ns] {
				continue
				}
				namespacesWaited[ns] = true
```

Reply:
Not unless you think that's a code smell. if continue implies else condition.